src/input/context.h

```diff
@@ -24,6 +24,7 @@
 #include <string>
 #include <vector>
 #include <unordered_map>
+#include <functional>
```

Comment:
didn't compile with my version of MSVC (and we use std::function in the code below)